_dev/eslint-plugin-fxa/lib/util/ast.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
```

Comment:
Not sure if this file needs a copyright header, or if it is related to the code comment above:
> ```js
> /*
> Borrowed from https://github.com/lo1tuma/eslint-plugin-mocha/tree/master/lib
> We cannot directly depend on that package due to lack of support for that in ESLint:  https://github.com/eslint/rfcs/pull/5
> */
> ```

Reply:
Heh I `cp`'d the files but didn't read every single one of them.  Um I guess we could link back to that repo in a comment in this file also.

Reply:
Yeah, I think the eslint-plugin-fxa plugin was after my time (I can't remember if we were originally using https://github.com/pdehaan/eslint-config-fxa at the time), so I just wasn't clear where the other files came from originally (or if that workaround is still needed for sharable configs, versus just installing whatever as a peer dependency).

Definitely not a merge blocker, just more of a curiosity since I'm a stickler for copyrights.